addons/web/static/src/core/utils/arrays.js

```diff
@@ -165,25 +165,6 @@ export function cartesian() {
     }
 }
 
-/**
- * Shallow compares two arrays.
- * @template T
- * @param {T[]} arrayA
- * @param {T[]} arrayB
- * @returns {boolean} true iff arrayA and arrayB are shallow equal
- */
-export function shallowEqual(arrayA, arrayB) {
```

Comment:
maybe keep it and call the other one. If I have two arrays and I want to shallowEqual them, I will certainly look into this file to see if there is an helper for this, not in objects.js